### PR TITLE
Clears up bad letters being used

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## Description
Replaces Ws with L and C for walls

## Motivation and Context
So we can really build the walls we want
## How Has This Been Tested?
This WILL work

## Screenshots (if appropriate):
n/a
## Changelog (necessary)
:cl:
fix: Corrects some wall building types from wooden barricades 
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
HUH Fun fact, moving well building is swapping what you build...

**Comment by Trilbyspaceclone** [comment]
As dumb as it is... I think this is better for the game to just allow all 3 types and you simply able to build all three by moving about. Merge after someone can conferm this to work tho.

**Comment by Laus4Deus** [comment]
smh
fine, I'll test it

**Comment by Laus4Deus** [comment]
But only because of how reliable you are

File: code/modules/fallout/obj/structures/obstacle.dm [hunk 1]
```diff
@@ -81,14 +81,14 @@
 					return TRUE
 			if(C.amount >= 3)
 				to_chat(user, "<span class='notice'>You start building a interio wall...</span>")
```
Comment (Laus4Deus) [inline code/modules/fallout/obj/structures/obstacle.dm:83]: change it to "you start building AN interioR wall..." xd

File: code/modules/fallout/obj/structures/obstacle.dm [hunk 2]
```diff
@@ -80,15 +80,15 @@
 					qdel(src)
 					return TRUE
 			if(C.amount >= 3)
-				to_chat(user, "<span class='notice'>You start building a interio wall...</span>")
-				if(do_after(user, 100, target = src) && W.use(3))
+				to_chat(user, "<span class='notice'>You start building an interio wall...</span>")
```
Comment (Laus4Deus) [inline code/modules/fallout/obj/structures/obstacle.dm:83]: xdddd an interio**R**

**Comment by Laus4Deus** [comment]
Adios
Anyway!!!! It works.
Clean compile, no errors and no bugs in game.

**Comment by Trilbyspaceclone** [comment]
@Laus4Deus Thx

**Pull request closed without merging**